cli/src/registry/registry.core.ts

```diff
@@ -1,11 +1,19 @@
+import { execSync } from 'child_process';
 import { RegistryEntry, SearchRegistryResult } from '../generated/control_panel';
 import { execAsync } from '../utils';
+import { Principal } from '@dfinity/principal';
 
 export enum Application {
   Station = 'station',
   Upgrader = 'upgrader',
 }
 
+export const getWasmChunkStoreId = (network: string = 'local'): Principal => {
+  const maybeCanisterId = execSync(`dfx canister id wasm_chunk_store --network ${network}`);
+
+  return Principal.fromText(maybeCanisterId.toString().split('\n').join(''));
```

Comment:
why split and join?

Reply:
Actually i can just use `trim` since in JS that also removes new lines.